src/test/java/de/orchound/gameoflife/model/BoardTest.java

```diff
@@ -18,11 +18,57 @@ void setUp() {
 	}
 
 	@Test
-	void testSetCellAlive() {
+	void testResurrectCellZeroCellIndex() {
 		board.resurrectCell(1, 0);
 		assertTrue(board.getCellStatus(1, 0));
 	}
 
+	@Test
+	void testResurrectCellMaxCellIndex() {
+		board.resurrectCell(1, 19);
+		assertTrue(board.getCellStatus(1, 19));
+	}
+
+	@Test
+	void testResurrectCellNegativeCellIndex() {
```

Comment:
Since I consider it the `Board` clients responsibility to provide valid indices, we don't need test methods like this at all.